maker-core/cvc.md

```diff
@@ -1,5 +1,40 @@
-# TBC
+# Constitutional Voter Committees (CVCs)
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Overview
+
+Constitutional Voter Committees (CVCs) are organized groups of MKR holders who have a shared vision . They form opinions on the direction of Maker Governance by producing and updating Position Documents.  
```

Comment:
```suggestion
Constitutional Voter Committees (CVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation that Maker Governance can use to have informed opinions on the CVC's vision.  
```

Reply:
Maybe : "They produce and publish documentation that Maker Governance can use to have informed opinions about the CVC's governance strategy." ?